drivers/tee/optee/core.c

```diff
@@ -431,40 +431,13 @@ optee_config_shm_memremap(optee_invoke_fn *invoke_fn, void **memremaped_shm,
 	unsigned long vaddr;
```

Comment:
With the restructuring of this function I think a better name would be `optee_config_shm_poolmgr()`